open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/mappers/graph/LineTypeMapper.java

```diff
@@ -25,11 +25,11 @@
      */
     static public String  mapLineTypeToRelationshipTypeGuid(LineType lineType) {
         String relationshipTypeName = lineType.name();
-        if (lineType.equals(LineType.Hasa)) {
+        if (lineType.equals(LineType.HasA)) {
             relationshipTypeName ="TermHASARelationship";
```

Comment:
I suggest changing this string to HasA so it matches the class name for consistency. Same for the other Lines.

Reply:
@davidradl I thought about this but I saw this docs https://egeria.odpi.org/open-metadata-publication/website/open-metadata-types/0380-Spine-Objects.html and I realized that in the repository the name is such

Reply:
@immmus I would suggest the omas is consistent with its own concepts; it should not use field names and the like because they match omrs, if there is a more appropriate concept to expose in the omas. This is one ofthe value adds of the omas, it exposes a more appropriate concept to a persona.